release/_git.sh

```diff
@@ -202,16 +204,9 @@ function create_branch {
 		original_branch_name="${LIFERAY_RELEASE_GIT_PREV_REF}"
 	fi
 
-	local last_commit=$(\
-		curl \
-			"https://api.github.com/repos/liferay/liferay-portal-ee/git/refs/heads/${original_branch_name}" \
-			--fail \
-			--header "Authorization: token ${LIFERAY_RELEASE_GITHUB_TOKEN}" \
-			--max-time 10 \
-			--retry 3 \
-			--silent)
+	local last_commit=$(invoke_get_github_api "https://api.github.com/repos/liferay/liferay-portal-ee/git/refs/heads/${original_branch_name}")
 
-	if [ $? -gt 0 ]
+	if [ $? == "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}" ]
```

Comment:
@kiwm, let's keep the consistency. Or we use `==` or `-eq`. You're using different operators to the exact same comparison here and in the line 225.